scripts/hyperkit.mjs

```diff
@@ -5,50 +5,111 @@ import os from 'os';
 import process from 'process';
 import childProcess from 'child_process';
 import path from 'path';
+import util from 'util';
 
 // The version of hyperkit to build
 const ver = 'v0.20210107';
 
+// Command lines for sudo.  Note that this will be passed to `sh -c`.
+const sudoTasks = [];
+
 /**
  * Build the Hyperkit binary.
- * @param workPath {string} The directory to work in; must be empty.
+ * @param destPath {string} The output path for the binary.
  * @returns {string} The executable in the work directory.
  */
-async function buildHyperkit(workPath) {
+async function buildHyperkit(destPath) {
+  try {
+    await fs.promises.access(destPath, fs.constants.X_OK);
+
+    const { gid } = await fs.promises.stat(destPath);
+
+    if (gid !== 80) {
+      console.log('hyperkit is owned by wrong group, rebuilding...');
+    } else {
+      console.log('hyperkit is acceptable, not building.');
+
+      return;
+    }
+  } catch (e) {
+    if (e.code !== 'ENOENT') {
+      console.error(e);
+    }
+    console.log('hyperkit not available, building...');
+  }
   // Using git and make to build the binary is intentional. There is no binary
   // download available from the project. Minikube checks the hyperkit version
```

Comment:
The comment is out-of-date as we no longer use minikube, but I think building hyperkit with proper version info is still the right idea anyways. I guess comment cleanup can happen at a later time when all other minikube references are being updated as well.